docs/source/data/fetching-rest.mdx

```diff
@@ -89,75 +126,88 @@ const resolvers = {
 
 ## Caching
 
-By default, data source implementations use Apollo Server's in-memory cache to store the results of past fetches.
+> 📣 **New in Apollo Server 4**: Apollo Server no longer automatically provides it's cache to data sources. [See here for more details](#datasources).
 
-When you initialize Apollo Server, you can provide its constructor a _different_ cache object that implements the [`KeyValueCache` interface](https://github.com/apollographql/apollo-utils/tree/main/packages/keyValueCache#keyvaluecache-interface). This enables you to back your cache with shared stores like Memcached or Redis.
+You can pass your `RESTDataSource` subclasses a cache (such as Apollo Server's default cache) to store the results of past fetches:
 
-### Using an external cache backend
+<MultiCodeBlock>
 
-When running multiple instances of your server, you should use a shared cache backend. This enables one server instance to use the cached result from _another_ instance.
+```ts
+import { RESTDataSource } from '@apollo/datasource-rest';
+// KeyValueCache is the type of Apollo server's default cache
+import { KeyValueCache } from '@apollo/utils.keyvaluecache';
```

Comment:
We _can_ use type imports for this (though I don't know that we do that consistently in our TS docs). This is totally optional.
```suggestion
import type { KeyValueCache } from '@apollo/utils.keyvaluecache';
```

Reply:
Oh, rad, I'll try to do that from now on!